utils/job_submitters/oar.py

```diff
@@ -87,7 +88,8 @@ def submit_job_and_monitor(self, commands):
             job_script.write(f"#OAR -n {self.job_name}\n")
             job_script.write(f"#OAR -l /nodes={self.nnode}/core={self.ppn},walltime={seconds_to_timestr(self.walltime)}\n")
             job_script.write(f"#OAR --project {self.project}\n")
-            job_script.write(f"#OAR -t {self.queue}\n")
+            if self.queue == 'devel':
```

Comment:
Oddly, `-t` argument means `type` in OAR system. This is different from partition/qos in slurm system for example, and it should be left blank for standard queues.